components/forms/Form/Form.tsx

```diff
@@ -19,15 +25,50 @@ declare global {
  * @param props
  */
 const InnerForm = (props: InnerFormProps & FormikProps<FormValues> & DynamicFormProps) => {
-  const { children, handleSubmit, t, isSubmitting } = props;
+  const { children, handleSubmit, t, isSubmitting, isReCaptchaEnableOnSite } = props;
   const [canFocusOnError, setCanFocusOnError] = useState(false);
   const [lastSubmitCount, setLastSubmitCount] = useState(0);
-
   const errorList = props.errors ? getErrorList(props) : null;
   const errorId = "gc-form-errors";
   const serverErrorId = `${errorId}-server`;
   const formStatusError = props.status === "Error" ? t("server-error") : null;
 
+  const handleFormSubmission = (evt: React.FormEvent<HTMLFormElement>) => {
```

Comment:
This function name is **very similar** to `handleSubmit`. It would be nice if it had a name that clearly indicated its function, highlighting that it is linked to `handleSubmit`, yet also highlighting how it's different. I would suggest `handleSubmitReCaptcha`.